htdocs/fourn/commande/dispatch.php

```diff
@@ -644,7 +644,7 @@
 
 		// Set $products_dispatched with qty dispatched for each product id
 		$products_dispatched = array();
-		$sql = "SELECT l.rowid, cfd.fk_product, sum(cfd.qty) as qty";
+		$sql = "SELECT l.rowid, cfd.fk_product, round(sum(cfd.qty), 2) as qty";
```

Comment:
rounding must never be done by the database because database has no knowledge of Dolibar rounding rules.
Instead the rounding must be done in PHP. It is done later by 
price2num($objd->qty, 5);
that should be
price2num($objd->qty, 'MS');